python/triton/compiler/compiler.py

```diff
@@ -46,11 +46,11 @@ def ttir_compute_capability_rewrite(mod, arch):
     pm.enable_debug()
     if _is_cuda(arch):
         pm.add_rewrite_tensor_pointer_pass(arch, False)
-    else:
-        device = get_current_device()
-        capabilityTuple = get_device_capability(device)
-        capability = capabilityTuple[0] * 10 + capabilityTuple[1]
+    elif is_hip():
+        capability = 90
```

Comment:
90 is referring H100.
The `capability` is only used [here](https://github.com/ROCmSoftwarePlatform/triton/blob/fa429316d4be5c8e18288dec127d63beda21843a/lib/Dialect/Triton/Transforms/RewriteTensorPointer.cpp#L473). So any number less than 90 should work.
And we used to put 80 [here](https://github.com/ROCmSoftwarePlatform/triton/blob/fa429316d4be5c8e18288dec127d63beda21843a/python/triton/compiler/compiler.py#L85)
So maybe also put 80 here?

Reply:
I think it is safe to put 80, but I think better solution is to introduce some more generic way to express target hardware. Compute capability number makes a little sense for AMD hardware.